<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkeletonWidget</class>
 <widget class="QWidget" name="SkeletonWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>660</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑 Light</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Sensitivity">
   <property name="geometry">
    <rect>
     <x>658</x>
     <y>140</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>灵敏度</string>
   </property>
   <widget class="QSlider" name="horizontalSliderSensitivity">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>50</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddSensitivity">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubSensitivity">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelSensitivity">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>horizontalSliderSensitivity</zorder>
   <zorder>btnAddSensitivity</zorder>
   <zorder>LabelSensitivity</zorder>
   <zorder>btnSubSensitivity</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_16">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>坐标微调</string>
   </property>
   <widget class="QPushButton" name="btnSkeletonDown">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>104</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkeletonRight">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkeletonLeft">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkeletonUp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>16</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Calc">
   <property name="geometry">
    <rect>
     <x>658</x>
     <y>380</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>快速计算</string>
   </property>
   <widget class="QCheckBox" name="checkBoxFast">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>快速计算使能</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_AngleRange">
   <property name="geometry">
    <rect>
     <x>658</x>
     <y>20</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>角度范围</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>237</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>角度范围:±</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAngle"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>641</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>区域</string>
   </property>
   <widget class="QRadioButton" name="radioButtonSkeletonStudyBit1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>学习区域:点1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSkeletonStudyBit2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>学习区域:点2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSkeletonFindBit1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索区域:点1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSkeletonFindBit2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>搜索区域:点2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStudy1">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>54</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(200,200)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStudy2">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>84</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(400,300)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFind1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>54</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(30,30)</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFind2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>84</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(610,450)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Score">
   <property name="geometry">
    <rect>
     <x>659</x>
     <y>260</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>相似度门限</string>
   </property>
   <widget class="QSlider" name="horizontalSliderScore">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>50</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddScore">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubScore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelScore">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
